CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to
 #### Added
 - Build time dependency on cereal
   - [#1893](https://github.com/iovisor/bpftrace/pull/1893)
+- Memory leak regression tests
+  - [#1920](https://github.com/iovisor/bpftrace/pull/1920)
```

Comment:
Changelog is for users to read. This change (while great!) should not go into changelog.

Reply:
Ok, I wasn't sure here, removed.